tests/snapstack_test.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
+from snapstack import Plan, Step
+
+class SnapstackTest(unittest.TestCase):
+
+    def test_snapstack(self):
+        '''
+        _test_snapstack_
+
+        Run a basic smoke test, utilizing our snapstack testing harness.
+
+        '''
```

Comment:
A comment here might be worthwhile along the lines mentioned in the keystone snap review, to mention what other default  base snaps are deployed alongside this snap and it's tests.